page-heading/main.scss

```diff
@@ -0,0 +1,27 @@
+@import 'o-colors/main';
+@import 'o-grid/main';
+@import 'o-typography/main';
+@include nUiMaybeCriticalStart('pageHeading');
+
+$o-page-heading-is-silent: not nUiHas('pageHeading') !default;
+
+@if nUiApply('pageHeading') {
+	// will move into origami
+	.n-page-heading-wrapper {
+		background: getColor('teal-1');
+		color: getColor('white');
+		margin-top: 0;
+		margin-bottom: $spacing-unit;
+	}
+	.n-page-heading {
+		@include oTypographySans(l);
```

Comment:
use the `$load-progressively: true` argument to turn on progressive loading of the font

https://github.com/Financial-Times/o-typography#progressively-loading-web-fonts